src/cbor.h

```diff
@@ -144,6 +148,7 @@ struct CborEncoder
         ptrdiff_t bytes_needed;
     };
     const uint8_t *end;
+    size_t added;
```

Comment:
This breaks ABI, is it OK? (Not sure about TinyCBOR's ABI compatibility promises.)